profiler/src/ProfilerEngine/Datadog.Profiler.Native/DebugInfoStore.h

```diff
@@ -42,10 +43,16 @@ class DebugInfoStore : public IDebugInfoStore
     template <typename TInfo>
     SymbolDebugInfo Get(TInfo& info, ModuleID moduleId, RID rid)
     {
-        if (!info.IsValid || rid >= info.SymbolsDebugInfo.size())
+        auto invalidInfo = !info.IsValid || rid >= info.SymbolsDebugInfo.size();
+
+        if (invalidInfo)
         {
-            Log::Info("The debug info for the module `", moduleId, "` seems to be invalid (is valid? '", info.IsValid, "') or the RID is out of the symbols array bounds (RID: ",
+            auto alreadyLogged = std::exchange(info.ErrorLogged, true);
+            if (!alreadyLogged)
+            {
+                Log::Info("The debug info for the module `", moduleId, "` seems to be invalid (is valid? '", info.IsValid, "') or the RID is out of the symbols array bounds (RID: ",
```

Comment:
- it could be easier to log one or the other case so the lines will be explicit what is wrong (invalid or RID)
- provide the corresponding filename to avoid having to find it elsewhere in the log file based on the moduleID value

Reply:
done in  4db30416eb34bb